docs/extend/extend-apiml/api-mediation-personal-access-token.md

```diff
@@ -0,0 +1,151 @@
+
+## Invalidate a specific token
+
+The user can invalidate the Personal Access Token by calling the following REST API endpoint via Gateway:
+
+`DELETE /auth/access-token/revoke`
+The full path of the `/auth/access-token/revoke` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/revoke`.
+
+The request requires the body in the following format:
+
+```json
+{
+  "token": "<token_value>"
+}
+```
+
+When the `/auth/access-token/revoke` endpoint is called, the provided PAT's hash will be stored in the Caching Service will be storing only hashes of already invalidated tokens. Access to these entries is protected by API ML’s client certificate.
+
+When invalidation is successful, the response to the request is an empty body.
+
+## Invalidate all the tokens for a user
+
+In case of possibility of breach, the Security Administrator can invalidate all the tokens based on some criteria.
+Such criteria define the level of access control and can restrict the access in advance. This can be done by using
+**rules**, and the access restriction can be applied by either user ID or service scopes.
+
+The administrator can invalidate all the tokens bound to a specific user by calling the following REST API endpoint via Gateway:
+
+`DELETE /auth/access-token/revoke/tokens/users`
+The full path of the `/auth/access-token/revoke/tokens/users` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/revoke/tokens/users`.
+
+The request requires the body in the following format:
+
+```json
+{
+   "userId": "<userId>",
+   "timestamp": "<timestamp>"
+}
+```
+
+The `userId` is the user the revocation should be applied to, while the timestamp represents the current date of revocation, in milliseconds. 
+
+When invalidation is successful, the response to the request is an empty body.
+
+## Invalidate all the tokens for a service
+
+As we mentioned in the above section, it is possible to revoke tokens also by using rules based on service scopes.
+The administrator can invalidate all the tokens bound to a specific service by calling the following REST API endpoint via Gateway:
+
+`DELETE /auth/access-token/revoke/tokens/scope`
+The full path of the `/auth/access-token/revoke/tokens/scope` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/revoke/tokens/scope`.
+
+The request requires the body in the following format:
+
+```json
+{
+   "serviceId": "<serviceId>",
+   "timestamp": "<timestamp>"
+}
+```
+
+The `serviceId` represents the service the revocation should be applied to (e.g. APPL IDs), while the timestamp represents the current date of revocation, in milliseconds.
+
+When invalidation is successful, the response to the request is an empty body.
+
+## Retrieve all the tokens and rules
+
+The user can retrieve all the invalidated tokens and rules which were stored in the cache by calling the following REST API endpoint via Gateway:
```

Comment:
Who has access to this endpoint?

Reply:
It's not valid anymore, as the user should not retrieve the tokens and rules. It should not be documented.